config-linux.md

```diff
@@ -154,15 +154,20 @@ In addition to any devices configured with this setting, the runtime MUST also s
 ## Control groups
 
 Also known as cgroups, they are used to restrict resource usage for a container and handle device access.
-cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
+cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` is expected to be relative to the cgroups mount point.
-If `cgroupsPath` is not specified, implementations can define the default cgroup path.
+If `cgroupsPath` is:
+* ... an absolute path (starting with `/`), the implementation MUST take the path to be relative to the cgroup mount point.
+* ... a relative path (not starting with `/`), implementations SHOULD take the value to be the suffix of the resulting cgroup path.
+* ... not specified, implementations CAN define the default cgroup path.
+Implementations CAN consider certain `cgroupsPath` values to be invalid, and MUST generate an error if this is the case.
+If a `cgroupsPath` value is specified, an implementation MUST consistently attach to the same place in the cgroup hierarchy given the same value of `cgroupsPath`.
+
 Implementations of the Spec can choose to name cgroups in any manner.
 The Spec does not include naming schema for cgroups.
-The Spec does not support [split hierarchy][cgroup-v2].
+The Spec does not support the [unified or "default" hierarchy][cgroup-v2].
```

Comment:
On Fri, Jun 10, 2016 at 11:45:55AM -0700, Michael Crosby wrote:
> > -The Spec does not support [split hierarchy][cgroup-v2].
> > +The Spec does not support the [unified or "default" hierarchy][cgroup-v2].
>
> is it important to have this?  Even if it was supported there would
> be no changes to the spec and this is more of a runtime concern
> right?

It was added to motivate a single cgroup-path property
vs. per-controller paths [1].  I think it's useful to keep the
(informative) motivation documented here (although I think we can
clarify the wording).

[1]: https://github.com/opencontainers/runtime-spec/pull/137#discussion_r38597501